oak-run-elastic/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/ElasticIndexer.java

```diff
@@ -64,6 +64,11 @@ public ElasticIndexer(IndexDefinition definition, FulltextBinaryTextExtractor bi
         this.indexHelper = indexHelper;
     }
 
+    @Override
+    public void onIndexingStarting() {
```

Comment:
Are we purposely leaving out the statistics logging for elastic?

Reply:
It was missing, indeed. I extracted the logic to keep track of the indexer statistics into a separate class and added this tracking to the ElasticIndexer as well.